graph/ThingGraph.java

```diff
@@ -214,15 +209,15 @@ public ThingVertex convertToReadable(VertexIID.Thing iid) {
     private <VAL, IID extends VertexIID.Attribute<VAL>, VERTEX extends AttributeVertex<VAL>> VERTEX getOrReadFromStorage(
             Map<IID, ? extends VERTEX> map, IID attIID, Function<IID, VERTEX> vertexConstructor) {
         VERTEX vertex = map.get(attIID);
-        if (vertex == null && storage.get(attIID.bytes()) != null) return vertexConstructor.apply(attIID);
+        if (vertex == null && storage.get(attIID) != null) return vertexConstructor.apply(attIID);
         else return vertex;
     }
 
     public FunctionalIterator.Sorted<ThingVertex> getReadable(TypeVertex typeVertex) {
-        ByteArray.Base prefix = join(typeVertex.iid().bytes(), Encoding.Edge.ISA.in().bytes());
+        Key.Prefix<EdgeIID.InwardsISA> prefix = EdgeIID.InwardsISA.prefix(typeVertex.iid());
```

Comment:
Each type of IID that we can look up in a partition can produce a `Key.Prefix` which can be iterated with. The prefix contains a builder for creating the full object